README.md

```diff
@@ -98,7 +98,8 @@ odometer = vehicle.odometer()['data']['distance']
 |429|smartcar.RateLimitingException|
 |430|smartcar.MonthlyLimitExceeded|
 |500|smartcar.ServerException|
-|501|smartcar.NotCapableException|
```

Comment:
Removing this is technically a breaking change, is it not? Might want to bump the big boy instead of the middle child.

Reply:
v valid point.